specification/containerregistry/resource-manager/Microsoft.ContainerRegistry/preview/2020-11-01-preview/containerregistry.json

```diff
@@ -0,0 +1,5633 @@
+          "enum": [
+            "Creating",
+            "Updating",
+            "Deleting",
+            "Succeeded",
+            "Failed",
+            "Canceled"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": true
+          }
+        },
+        "mode": {
+          "description": "The mode of the connected registry resource that indicates the permissions of the registry.",
+          "enum": [
+            "Registry",
+            "Mirror"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "ConnectedRegistryMode",
+            "modelAsString": true
+          }
+        },
+        "version": {
+          "description": "The current version of ACR runtime on the connected registry.",
+          "type": "string",
+          "readOnly": true
+        },
+        "lastVersionUpdateTime": {
+          "format": "date-time",
+          "description": "The last time that the ACR runtime version was updated on the connected registry.",
+          "type": "string",
+          "readOnly": true
+        },
+        "connectionState": {
+          "description": "The current connection state of the connected registry.",
+          "enum": [
+            "Online",
+            "Offline",
+            "Syncing",
+            "Unhealthy"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ConnectionState",
+            "modelAsString": true
+          }
+        },
+        "lastActivityTime": {
+          "format": "date-time",
+          "description": "The last activity time of the connected registry.",
+          "type": "string",
+          "readOnly": true
+        },
+        "parent": {
+          "$ref": "#/definitions/ParentProperties",
+          "description": "The parent of the connected registry."
+        },
+        "clientTokenIds": {
+          "description": "The list of the ACR token resource IDs used to authenticate clients to the connected registry.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "loginServer": {
+          "$ref": "#/definitions/LoginServerProperties",
+          "description": "The login server properties of the connected registry."
+        },
+        "logging": {
+          "$ref": "#/definitions/LoggingProperties",
+          "description": "The logging properties of the connected registry."
+        },
+        "statusDetails": {
+          "description": "The list of current statuses of the connected registry.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/StatusDetailProperties"
+          },
+          "readOnly": true
+        }
+      }
+    },
+    "ParentProperties": {
+      "description": "The properties of the connected registry parent.",
+      "required": [
```

Comment:
should id property be added here @jaysterp, its being generated locally

Reply:
It is being removed from dev.